ui.content/src/main/content/jcr_root/content/experience-fragments/venia/.content.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     cq:adobeTargetExportFormat="html"
-    cq:allowedTemplates="[/conf/venia/settings/wcm/templates/xf-web-variation]"
     cq:conf="/conf/venia"
+    cq:allowedTemplates="[/conf/venia/settings/wcm/templates/(?!page-).*]"
```

Comment:
Is this correct? I mean, the only allowed template should be the XF one, no?

Reply:
Fixed it